package.json

```diff
@@ -53,6 +53,8 @@
     "react-dom": "^16.4.1",
     "react-helmet": "^5.2.0",
     "react-router-dom": "^4.3.1",
+    "remark": "^9.0.0",
+    "remark-react": "^4.0.3",
```

Comment:
As the issue didn't say which packages to use, can we get a clear picture of which are good to use and what else was considered in the PR comments please.